tests/functional/Generator/PhpStormMetaTest.php

```diff
@@ -303,11 +303,15 @@ private function assertSettings(): void {
     $generated_content = $this->getGeneratedContent('.phpstorm.meta.php/settings.php');
     // The full list of settings depends on environment.
     $settings = <<< 'PHP'
+        'hash_salt',
+        'update_free_access',
         'container_yamls',
         'file_scan_ignore_directories',
         'entity_update_batch_size',
         'entity_update_backup',
+        'state_cache',
```

Comment:
https://www.drupal.org/project/drupal/issues/2575105

Something else led to these changes, I didn't find it.